ui-components/menuitems.md

```diff
@@ -122,7 +122,7 @@ editor.ui.registry.addNestedMenuItem('nesteditem', {
 
 ### Toggle menu items
 
-A toggle menu item triggers its `onAction` when clicked. It also has a concept of state. This means it can be toggled `on` and `off`. A toggle menu item gives the user visual feedback for its state through a checkmark that appears to the left of the menu item's text when it is `on`.
+A toggle menu item triggers its `onAction` when clicked. It also has a concept of state. This means it can be toggled `on` and `off`. A toggle menu item gives the user visual feedback for its state through a checkmark that appears to the right of the menu item's text when it is `on`.
```

Comment:
The config options table below will have to be update to include `icon` as an option. Additionally, the `Note:` at the bottom of the table will have to be updated to say this has changed in 5.3.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1486/commits/2a7723745c44125c34fdfddb8a9ccbd53df8bbbb